app/views/doc/ext.html.erb

```diff
@@ -90,6 +90,13 @@
   <div class='two-column'>
     <div class='column-left'>
       <ul class='books-list'>
+        <li>
```

Comment:
please, don't add your book to the top of the list. While there's no explicit ordering, it seems unlikely to have some other book than "Pro Git" on the top of the list